content/en/news/releases/1.6.x/announcing-1.6.x/upgrade-notes/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Upgrade Notes
+description: Important Changes to consider when upgrading to Istio 1.6.
```

Comment:
```suggestion
description: Important changes to consider when upgrading to Istio 1.6.
```

Reply:
This file will be removed before merging. Netlify wouldn't render the docs without this.